grpc/thread_pool.cpp

```diff
@@ -0,0 +1,29 @@
+#include "thread_pool.hpp"
+
+#include "job.hpp"
+
+namespace ioremap { namespace grpc {
+
+`thread_pool_t::thread_pool_t(::grpc::CompletionQueue &completion_queue, std::size_t size)
+: completion_queue_(completion_queue) {
+    auto loop = [this] {
```

Comment:
Whole file bad indent. Better to set in IDE settings tab_size = 8, otherwise you'll have this problem as usual.

Reply:
006d783